### PR TITLE
Add - adding new bouncing animation #639

### LINKED ISSUE #9: Create your own hacktoberfest animation
- Fork this repo
- Create a new CSS file with your cool animation name inside `./animations` folder, be creative!
- Edit the `./animations.js` file to include your animation using the following structure:

```js
{
    name: 'the-name-of-your-animation',
    author: 'your-github-username'
}
```

<sub>**PS**: the animation name should have the same name of your CSS file.</sub><br>
<sub>**PS²**: although it is not necessary to include your object alphabetically (there is a function that will do this automatically afterwards), it is recommended that you do so to keep the project organized.</sub><br>
<sub>**PS³**: use the same name of your CSS file as your animation class. Example: a file named `my-cool-animation.css` containing a `.my-cool-animation` class inside.</sub>

- Submit a merge request

- DONE!

If you have any problems/questions or issues please submit an issue and I will get to you!

Don't forget to star!

### LINKED ISSUE #639: Hacktoberfest 2023
- Fork this repo
- Create a new CSS file with your cool animation name inside `./animations` folder, be creative!
- Edit the `./animations.js` file to include your animation using the following structure:

```js
{
    name: 'the-name-of-your-animation',
    author: 'your-github-username'
}
```

<sub>**PS**: the animation name should have the same name of your CSS file.</sub><br>
<sub>**PS²**: although it is not necessary to include your object alphabetically (there is a function that will do this automatically afterwards), it is recommended that you do so to keep the project organized.</sub><br>
<sub>**PS³**: use the same name of your CSS file as your animation class. Example: a file named `my-cool-animation.css` containing a `.my-cool-animation` class inside.</sub>

- Submit a merge request

- DONE!

You don't need to be assigned to this issue to participate.

If you have any problems/questions or issues please submit an issue and I will get to you!

Don't forget to star!

### PR DESCRIPTION
**Pull request by 0nikhilkumar**
Adding new bouncing reflection animation

Fixes #639   

newRefelectionBouncing

Type of change
--adding bouncing reflection animation wave from top to bottom

How should this be tested?
1. Go on home page
2. Select newBouncingRefelection. 0nikhilkumar

### CODE REVIEW

File: animations/newBouncingRefelection.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  background-color:  #151719;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
```
Comment (NiallEccles) [inline animations/newBouncingRefelection.css:17]: Can you move these into your class?

File: animations/newBouncingRefelection.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.waviy {
```
Comment (NiallEccles) [inline animations/newBouncingRefelection.css:18]: This class name needs to match your animation name

**Comment by 0nikhilkumar** [comment]
Sir Some changes are done.

Sir I am hacktoberfest'23 participants so please merge this Pull Request and give the label of "hacktoberfest-accepted" then its counted for merging 1PR in hacktoberfest'23 event. @NiallEccles

Thank you sir.

**Comment by 0nikhilkumar** [comment]
Sir Changes are done please review it and merge it and please give the label "hacktoberfest-accepted" @NiallEccles

**Pull request merged**